templates/devices/device_detail.html

```diff
@@ -168,7 +168,11 @@
                 {% if device.manual %}
                     <tr>
                     <td>{% trans "Manual" %}</td>
-                    <td><a href="{{ device.manual.url }}">{{ device.manual|filename }}</a></td></tr>
+                    <td><a href="{{ device.manual.url }}">
+                      {%with filename=device.manual|filename|splitstr:'.'|last %}
+                      {{device.name|addstr:'.'|addstr:filename}}
```

Comment:
`{{ device.name ~ '.' ~ filename }}` is not working?

http://jinja.pocoo.org/docs/2.10/templates/#other-operators

Reply:
I didn't know we used Jinja.

Reply:
Also, apparently, we don't. Is it really worth changing the templating language, when a simple template tag would do the trick, too? I don"t even know, why jinja is in the installed packages, I couldn't find any place where it was used.

Reply:
touché :innocent: 

But even with DTL you can use [add](https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#add) 

>This filter will first try to coerce both values to integers. If this fails, it’ll attempt to add the values together anyway. This will work on some data types (strings, list, etc.) and fail on others. If it fails, the result will be an empty string.

Reply:
"Warning:
Strings that can be coerced to integers will be summed, not concatenated, as in the first example above."
Too risky, right?